libs/ledger-live-common/src/families/near/speculos-deviceActions.ts

```diff
@@ -0,0 +1,38 @@
+import type { DeviceAction } from "../../bot/types";
+import type { Transaction } from "./types";
+import { deviceActionFlow } from "../../bot/specs";
+
+export const acceptTransaction: DeviceAction<Transaction, any> =
+  deviceActionFlow({
+    steps: [
+      {
+        title: "Confirm",
+        button: "Rr",
+      },
+      {
+        title: "Deposit",
+        button: "Rr",
+      },
+      {
+        title: "Amount",
+        button: "Rr",
+      },
+      {
+        title: "To",
+        button: "Rr",
+      },
+      {
+        title: "From",
+        button: "Rr",
+      },
+      {
+        title: "Args",
+        button: "Rr",
+      },
```

Comment:
this is ok as a "MvP" but it would be create to add "expectedValue" to add more checks on the "expected to display on screen" parts

Reply:
That makes sense - thx for marking it a non-blocker, I'll give a think about how to best handle it in NEAR's case and if it doesn't make it into this PR I can include it in a follow-up.

Reply:
Addressed here: https://github.com/LedgerHQ/ledger-live/pull/1126/commits/ffb55d2005a02e2331faf03d0731382036725d80